partiql-lang/build.gradle.kts

```diff
@@ -64,6 +64,24 @@ dependencies {
     }
 }
 
+val relocations = mapOf(
+    "org.antlr" to "org.partiql.thirdparty.antlr"
```

Comment:
I had thought about this issue previously and tested locally. Seems like there's no collision:

1. Defining a project w/ just the `partiql-lang-kotlin` dependency, `org.partiql.thirdparty.antlr.v4.runtime.Token` would resolve to use `partiql-lang-kotlin`'s definition.
2. Defining a project w/ `partiql-lang-kotlin` and `partiql-parser` dependency (in both orderings in the build.gradle.kts file), `org.partiql.thirdparty.antlr.v4.runtime.Token` resolves to use `partiql-lang-kotlin`'s defintion.

Based on above, it seems to always resolve to the `lang` dependency.

Not sure if there's a use-case in which we would have different versions of ANTLR for `lang` and `parser` since the `lang` dependency should match the generated `parser`'s dependency. To prevent any potential issues w/ namespacing, I'll add an additional package suffix before `thirdparty`.

Reply:
With the above change to namespacing, users can be sure where the ANTLR runtime dependency is coming from:

```
import org.partiql.lang.thirdparty.antlr.v4.runtime.Token as PartiQLParserToken
import org.partiql.parser.thirdparty.antlr.v4.runtime.Token as PartiQLLangToken
```